fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4980,13 +4983,18 @@ class FMCMainDisplay extends BaseAirliners {
         return 0 <= fuel && fuel <= 80;
     }
 
-    /**
-     *
-     * @returns {*}
+    /**NOTE: This is essentially the 'top level/master' injection point for the current on board fuel.
+     *ALL fuel projections, current fuel level displays etc use the figure returned here as the baseline.
+     * @returns {either the detected fuel in the tanks(if at least one engine running) or pilot entered block fuel(engines off)}
      */
     //TODO: Can this be util?
     getFOB() {
-        return (SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "pound") * 0.4535934) / 1000;
+        if (this.isAnEngineOn()) {
+            return (SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "pound") * 0.4535934) / 1000;
+        } else {
+            //If an engine is not running, use pilot entered block fuel to calculate fuel predictions
+            return this.blockFuel * (0.4535934 / 1000);
+        }
```

Comment:
`blockFuel` is already in tons so we shouldn't try to apply a pounds to tons conversion to it.
Note: this.blockFuel is undefined unless entered, so would have returned NaN before. undefined is better and consistent with how we handle this elsewhere.
```suggestion
        const useFqi = this.isAnEngineOn();
        if (useFqi) {
            return (SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "pound") * 0.4535934) / 1000;
        } else {
            // If an engine is not running, use pilot entered block fuel to calculate fuel predictions.
            return this.blockFuel;
        }
```

Reply:
see below regarding changing the logic here.  I did remove the extra conversion though.(in local)